Dockerfile.base

```diff
@@ -134,7 +134,7 @@ COPY --from=cache --chown=$UID /calypso/composer.* /calypso/
 RUN apt update &&\
 	apt-get install -y apt-transport-https zip &&\
 	wget https://packages.sury.org/php/apt.gpg -O /etc/apt/trusted.gpg.d/php-sury.gpg &&\
-	echo "deb https://packages.sury.org/php/ stretch main" > /etc/apt/sources.list.d/php-sury.list &&\
+	echo "deb https://packages.sury.org/php/ buster main" > /etc/apt/sources.list.d/php-sury.list &&\
```

Comment:
Without this change, the docker build will fail, so we should merge this PR today